scripts/quests/bastok/Shady_Business.lua

```diff
@@ -30,11 +30,11 @@ quest.sections =
 
         [xi.zone.PORT_BASTOK] =
         {
-            ['Talib'] = quest:progressEvent(2),
+            ['Talib'] = quest:progressEvent(90),
```

Comment:
This is confirmed and tested to be correct?

Reply:
Yep.  90 was the ID being used in Talib's personal script and I looked through the CS mine for Port Bastok to make sure before changing.